src/services/plugins.js

```diff
@@ -0,0 +1,23 @@
+import store from '../vuex/store'
+import * as pluginMutations from '../vuex/modules/plugins/mutation-types'
+
+const pluginList = [
+  'dummy'
+]
+
+export default {
+  loadAll () {
+    pluginList.forEach((pluginName) => {
+      this.loadOne(pluginName)
+    })
+  },
+  loadOne (pluginName) {
+    // require(['../../node_modules/kuzzle-bo-plugin-' + pluginName + '/index'], (plugin) => {
+    require(['../../plugins/kuzzle-bo-plugin-' + pluginName + '/index'], (plugin) => {
```

Comment:
It's a way to avoid that Webpack builds the whole `node_modules` (in case the plugins are located there, which is something we should discuss)